nephio-5gc-controller/apis/nf/v1alpha1/upf_types.go

```diff
@@ -20,66 +20,66 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type UpfN3 struct {
+type UPFN3 struct {
 	Endpoints []Endpoint `json:"endpoints"`
 }
 
-type UpfN4 struct {
+type UPFN4 struct {
 	Endpoints []Endpoint `json:"endpoints"`
 }
 
 type N6Endpoint struct {
-	Dnn         string   `json:"dnn"`
-	IpEndpoints Endpoint `json:"ipendpoints"`
+	DNN         string   `json:"dnn"`
```

Comment:
Should we spell this out?

Reply:
Data Network Name  --- putting it in comment should suffice